x/evm/types/params.go

```diff
@@ -27,8 +27,8 @@ var (
 	DefaultEVMDenom = utils.BaseDenom
 	// DefaultAllowUnprotectedTxs rejects all unprotected txs (i.e false)
 	DefaultAllowUnprotectedTxs = false
-	// AvailableEVMExtensions defines the default active precompiles
-	AvailableEVMExtensions = []string{
+	// AvailableStaticPrecompiles defines the default active precompiles
```

Comment:
This is not clear. Available means that they are present, enabled or not enabled. Default means the one that are active? I believe we should discriminate between what can be enabled and what is enabled by default, similarly to the EIPs.